javascriptv3/Dockerfile

```diff
@@ -3,6 +3,5 @@
 # GA updates: https://github.com/awsdocs/aws-doc-sdk-examples/issues/4127
 FROM node:18
 COPY . /javascriptv3
-RUN cd /javascriptv3 && \
-    npm install
-CMD ["bash"]
+RUN npm install --prefix /javascriptv3
+CMD ["bash", "-c", "cd /javascriptv3 && bash"]
```

Comment:
Why?

Reply:
`CMD ["bash"]` tells Docker to start a bash session.
I needed to change to the javascriptv3 directory, so `CMD ["bash", "-c", "cd /javascriptv3"]` changes to the right directory.
However, just `bash -c` itself doesn't start a bash session, hence `&& bash`.

Reply:
Why do you need to start a bash session from within your javascriptv3 directory?

Reply:
After our conversation, I've reverted this to just `CMD ["bash"]`